README.md

```diff
@@ -1,12 +1,200 @@
+        redshiftClusterMaintenanceSettingsCheck: { preferredMaintenanceWindow: "Mon:09:30-Mon:10:00" },
+        acmCheckCertificateExpiration: { maxDaysUntilExpiration: 10 },
+    });
+    ```
+
+### Test the new Policy Pack
+
+Policy Packs can be tested on a user's local workstation to facilitate rapid development and testing of policies.
+
+1. Run `npm install` in the Policy Pack directory.
+
+1. Use the `--policy-pack` flag with `pulumi preview` or `pulumi up` to specify the path to the directory containing your Policy Pack when previewing/updating a Pulumi program.
+
+    If you don’t have a Pulumi program readily available, you can create a new project for testing by running `pulumi new aws-typescript` in an empty directory. This AWS example will create an S3 bucket, which is perfect for testing our Policy.
+
+    In the Pulumi project's directory run:
+
+    **macOS or Linux**:
+
+    ```sh
+    PULUMI_DEBUG_COMMANDS=true pulumi preview --policy-pack <path-to-policy-pack-directory>
+    ```
+
+    **Windows cmd.exe**
+
+    ```sh
+    set PULUMI_EXPERIMENTAL=true
+    pulumi preview --policy-pack <path-to-policy-pack-directory>
+    ```
+
+    **Windows PowerShell**
+
+    ```sh
+    $env:PULUMI_EXPERIMENTAL = 'true'
+    pulumi preview --policy-pack <path-to-policy-pack-directory>
+    ```
+
+    If the stack is not in compliance, the policy violation will be displayed. Since the enforcement level for all policies are set to advisory, a warning is shown for any resources that are not in compliance with the AWSGuard policies. In this case, logging must be defined for S3 buckets.
+
+    ```
+    Previewing update (dev):
+
+        Type                 Name           Plan       Info
+    +   pulumi:pulumi:Stack  test-dev       create
+    +   └─ aws:s3:Bucket     my-bucket      create     1 warning
+
+    Diagnostics:
+    aws:s3:Bucket (my-bucket):
+        advisory: [s3-bucket-logging-enabled] Checks whether logging is enabled for your S3 buckets.
+        Bucket logging must be defined.
+
+    Resources:
+        + 2 to create
+    ```
+
+1. If you had wanted the preview to fail for any policy violations, the Policy Pack can be modified to configure all policies to be mandatory.
+
+    ```typescript
+    new AwsGuard({ all: "mandatory" });
+    ```
+
+1. Running the `pulumi preview` command again will now fail the preview operation.
+
+    ```
+    Previewing update (dev):
+
+        Type                 Name           Plan       Info
+    +   pulumi:pulumi:Stack  test-dev       create     1 error
+    +   └─ aws:s3:Bucket     my-bucket      create     1 error
+
+    Diagnostics:
+    pulumi:pulumi:Stack (test-dev):
+        error: preview failed
+
+    aws:s3:Bucket (my-bucket):
+        mandatory: [s3-bucket-logging-enabled] Checks whether logging is enabled for your S3 buckets.
+        Bucket logging must be defined.
+    ```
+
+1. If you do not want to enforce this particular policy, you can modify the Policy Pack to disable it.
```

Comment:
What I really wanted to have here was a way to fix the Pulumi program to be in compliance of the policy. But given the way we've currently implemented the `s3-bucket-logging-enabled` rule, I don't think it's easily possible (at least not all from within the same Pulumi program). I've opened #36 to track fixing this.